packages/p2p-v2/src/components/Verification/Verification.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Loader } from '@deriv-com/ui/dist/components/Loader';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useDevice, usePoiPoaStatus } from '../../hooks';
+import SendEmailIcon from '../../public/ic-send-email.svg';
+import { Checklist } from '../Checklist';
+import './Verification.scss';
+
+const getPoiAction = (status: string | undefined) => {
+    switch (status) {
+        case 'pending':
+            return 'Identity verification in progress.';
+        case 'rejected':
+            return 'Identity verification failed. Please try again.';
+        case 'verified':
+            return 'Identity verification complete.';
+        default:
+            return 'Upload documents to verify your identity.';
+    }
+};
+
+const getPoaAction = (status: string | undefined) => {
+    switch (status) {
+        case 'pending':
+            return 'Address verification in progress.';
+        case 'rejected':
+            return 'Address verification failed. Please try again.';
+        case 'verified':
+            return 'Address verification complete.';
+        default:
+            return 'Upload documents to verify your address.';
+    }
+};
+
+const Verification = () => {
+    const { isMobile } = useDevice();
+    const { data, isLoading } = usePoiPoaStatus();
+    const { isP2PPoaRequired, isPoaPending, isPoaVerified, isPoiPending, isPoiVerified, poaStatus, poiStatus } =
+        data || {};
+
+    const redirectToVerification = (route: string) => {
+        const search = window.location.search;
+        let updatedUrl = `${route}?ext_platform_url=/cashier/p2p`;
+
+        if (search) {
+            const urlParams = new URLSearchParams(search);
+            const updatedUrlParams = new URLSearchParams(updatedUrl);
+            urlParams.forEach((value, key) => updatedUrlParams.append(key, value));
+            updatedUrl = `${updatedUrl}&${urlParams.toString()}`;
+        }
+        window.location.href = updatedUrl;
```

Comment:
```suggestion
        window.location.href = updatedUrl;
```
@ameerul-deriv, why not use `history.push(updatedUrl);` here? Is a full page reload desired?

Reply:
ah yeah, better we use history.push instead